packages/application-extension/src/index.ts

```diff
@@ -262,18 +269,22 @@ const pages: JupyterFrontEndPlugin<void> = {
         window.open(`${baseUrl}lab`);
       }
     });
+    const page = PageConfig.getOption('notebookPage');
 
     app.commands.addCommand(CommandIDs.openTree, {
-      label: trans.__('Open Files'),
+      label: trans.__('File Browser'),
       execute: () => {
-        window.open(`${baseUrl}tree`);
+        if (page === 'tree') {
+          app.commands.execute(filebrowserCommandIDs.activate);
```

Comment:
To avoid bringing a dependency on `@jupyter-notebook/tree` for `@jupyter-notebook/application-extension`, maybe we should just use the `filebrowser:activate` string here directly?

Reply:
For example this is what is done in the JupyterLab `docmanager-extension`, using a command defined in another plugin: https://github.com/jupyterlab/jupyterlab/blob/d614e293eb4b90055307828d5e81c9661e1147c9/packages/docmanager-extension/src/index.tsx#L1125

Reply:
Sure, perhaps we should do the same in `tree-extension:openFileBrowser` and drop the `CommandIDs` namespace in `tree`.